### ISSUE TITLE
"Invalid Argument" on "new URLSearchParams(formData.entries())" in Edge 44.18362.449.0

### ISSUE THREAD

**Issue by FagnerMartinsBrack** [lifaon74/url-polyfill#56]
If you use spread, it works:

```javascript
const formData = new FormData(event.target);
alert(new URLSearchParams([...formData.entries()]).toString());
```

Here's an SSCCE that fails on Edge: https://jsfiddle.net/f7uxne19/

The reason is outlined [here](https://stackoverflow.com/questions/58117132/window-urlsearchparams-broken-on-edge#comment107498034_58118581):

> The problem in Edge is that they do not support destructuring of FormData object. (for ... of in linked table).

**Comment by lifaon74** [comment]
Yes, this is a legit behavior: the introduction of Symbol.iterable is pretty recent, just like classes/functions accepting them as input. This polyfill currently doesn't support iterables as input to support IE. But this could be easily solved by using `iteratorSupported` in the constructor.

I see 2 solutions for you:

1) `new URLSearchParams(Array.from(formData.entries())) `
2) Do a pull request which solves this problem

**Comment by FagnerMartinsBrack** [comment]
@lifaon74 Thanks for the response :)

I'm trying to run the tests locally to create a failing test and eventually a PR. However, there are a lot of moving parts. Do you know a step by step on how to run the tests for EDGE on Mac? If not that's fine I'll spend some time to try to figure this out.

**Comment by lifaon74** [comment]
The test are currently running on my own selenium test server. You may change the url here: https://github.com/lifaon74/url-polyfill/blob/master/tests/config.json

You may try to update the code, verify if everything works on your side, then do a pull request. I'll run the tests myself for your PR.
